ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh

```diff
@@ -46,36 +44,42 @@ cat << EOF > ~/ocp-install.yml
     ignore_errors: yes
   - name: Remove lock file
     file:
-      path: /home/tester/vm_ready.txt
+      path: /home/kni/vm_ready.txt
       state: absent
   - name: Run deployment
-    shell: kcli create plan --skippre --paramfile /home/tester/kcli_parameters.yml upstream_ci $KCLI_PARAM
+    shell: kcli create plan --skippre --paramfile /home/kni/kcli_parameters.yml upstream_ci $KCLI_PARAM
     args:
       chdir: ~/kcli-openshift4-baremetal
 EOF
+
 cat << EOF > ~/copy-kubeconfig-to-bastion.yml
----
-- name: Copy kubeconfig from installer to bastion
+- name: Copy kubeconfig from installer to vm
   hosts: all
   tasks:
-  - name: Run playbook to copy kubeconfig from installer vm to bastion vm
-    shell: ansible-playbook -i /home/tester/inventory /home/tester/kubeconfig.yml
+    - name: Copy kubeconfig from installer vm
+      shell: kcli scp root@cnfdc5-installer:/root/ocp/auth/kubeconfig /home/kni/.kube/config
+    - name: Add skip-tls-verify to kubeconfig
+      lineinfile:
+        path: /home/kni/.kube/config
+        regexp: '    certificate-authority-data:'
+        line: '    insecure-skip-tls-verify: true'
 EOF
+
 cat << EOF > ~/fetch-kubeconfig.yml
 ---
 - name: Fetch kubeconfig for cluster
   hosts: all
   tasks:
   - name: Grab the kubeconfig
     fetch:
-      src: /home/tester/.kube/config
+      src: /home/kni/.kube/config
       dest: $SHARED_DIR/kubeconfig
       flat: yes
   - name: Modify local copy of kubeconfig
     lineinfile:
       path: $SHARED_DIR/kubeconfig
-      regexp: '    server: https://127.0.0.1:6443'
-      line: "    server: https://sshd.bastion-telco:6443"
+      regexp: '    server: https://api.cnfdc5.t5g.lab.eng.bos.redhat.com:6443'
```

Comment:
same for the api and ip here, maybe keeping it in a var? so it will be easy to change if needed